### DISCUSSION TITLE
Hardware transcoding: Plex is working, Jellyfin not workig

### DISCUSSION THREAD

**Discussion by nicoska84** [tteck/Proxmox#2543] [Help Needed]
Hi,
I installed from fresh both Plex and Jellyfin. With Plex I'm able to use hardware transcoding without any issue.
With Jellyfin, even if I choose "hardware transcoding" in the Dashboard I still see huge CPU cores usage.

Do you know what could it be?
Thank you
Nico

**Comment by tteck** [comment]
Some tools to help
```
apt-get -y install {vainfo,intel-gpu-tools}
```
command `vainfo`
command `intel_gpu_top`

**Reply by nicoska84** [reply]
vainfo output:

`root@jellyfin:~# vainfo 
error: can't connect to X server!
libva info: VA-API version 1.14.0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_14
libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.14 (libva 2.12.0)
vainfo: Driver version: Intel iHD driver for Intel(R) Gen Graphics - 22.3.1 ()
vainfo: Supported profile and entrypoints
      VAProfileNone                   : VAEntrypointVideoProc
      VAProfileNone                   : VAEntrypointStats
      VAProfileMPEG2Simple            : VAEntrypointVLD
      VAProfileMPEG2Main              : VAEntrypointVLD
      VAProfileH264Main               : VAEntrypointVLD
      VAProfileH264Main               : VAEntrypointEncSliceLP
      VAProfileH264High               : VAEntrypointVLD
      VAProfileH264High               : VAEntrypointEncSliceLP
      VAProfileJPEGBaseline           : VAEntrypointVLD
      VAProfileJPEGBaseline           : VAEntrypointEncPicture
      VAProfileH264ConstrainedBaseline: VAEntrypointVLD
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSliceLP
      VAProfileVP8Version0_3          : VAEntrypointVLD
      VAProfileHEVCMain               : VAEntrypointVLD
      VAProfileHEVCMain               : VAEntrypointEncSliceLP
      VAProfileHEVCMain10             : VAEntrypointVLD
      VAProfileHEVCMain10             : VAEntrypointEncSliceLP
      VAProfileVP9Profile0            : VAEntrypointVLD
      VAProfileVP9Profile0            : VAEntrypointEncSliceLP
      VAProfileVP9Profile1            : VAEntrypointVLD
      VAProfileVP9Profile1            : VAEntrypointEncSliceLP
      VAProfileVP9Profile2            : VAEntrypointVLD
      VAProfileVP9Profile2            : VAEntrypointEncSliceLP
      VAProfileVP9Profile3            : VAEntrypointVLD
      VAProfileVP9Profile3            : VAEntrypointEncSliceLP
      VAProfileHEVCMain12             : VAEntrypointVLD
      VAProfileHEVCMain422_10         : VAEntrypointVLD
      VAProfileHEVCMain422_12         : VAEntrypointVLD
      VAProfileHEVCMain444            : VAEntrypointVLD
      VAProfileHEVCMain444            : VAEntrypointEncSliceLP
      VAProfileHEVCMain444_10         : VAEntrypointVLD
      VAProfileHEVCMain444_10         : VAEntrypointEncSliceLP
      VAProfileHEVCMain444_12         : VAEntrypointVLD
      VAProfileHEVCSccMain            : VAEntrypointVLD
      VAProfileHEVCSccMain            : VAEntrypointEncSliceLP
      VAProfileHEVCSccMain10          : VAEntrypointVLD
      VAProfileHEVCSccMain10          : VAEntrypointEncSliceLP
      VAProfileHEVCSccMain444         : VAEntrypointVLD
      VAProfileHEVCSccMain444         : VAEntrypointEncSliceLP
      VAProfileAV1Profile0            : VAEntrypointVLD
      VAProfileHEVCSccMain444_10      : VAEntrypointVLD
      VAProfileHEVCSccMain444_10      : VAEntrypointEncSliceLP`

intel_gpu_top (lunched when playing something on Jellyfin):

![image](https://github.com/tteck/Proxmox/assets/73702791/84a3e38e-e6ab-4daf-bd73-70e637fa8afd)

**Comment by nicoska84** [comment]
So, after updating to latest Proxmox Kernel, here is the situation (playing the same file, transcoded at the same quality):

Plex LXC (~1% cpu)
![image](https://github.com/tteck/Proxmox/assets/73702791/49ffb9d6-0db7-4dc3-a6a3-d64d73c860c5)

Jellyfin LXC (~15% cpu)
![image](https://github.com/tteck/Proxmox/assets/73702791/4420026e-00c1-4dc6-8c2a-7f066a570141)


Container got the same numbers of cores and RAM.

Pretty strange, maybe Plex transcoder is better?

**Reply by tteck** [reply]
Plex for the win, for sure.

**Reply by nicoska84** [reply]
The last thing is to try Tdarr and see

**Reply by nicoska84** [reply]
Nope, Tdarr is not converting using GPU :/

**Reply by nicoska84** [reply]
With this solution, seems to work: https://github.com/tteck/Proxmox/discussions/1765#discussioncomment-8565530
But CPU usage is increasing anyway.
